### PR TITLE
Android Toolset Translation (German)

### PR DESCRIPTION
**Pull request by Laserlicht**
German Translation of Toolset App.

If anyone wants to contribute other languages: feel free to attach xml file, then I'll expand the PR.

@oleg-derevenetz It should be enough to add the file, or is there more to be done?

### CODE REVIEW

**Comment by oleg-derevenetz** [comment]
> It should be enough to add the file, or is there more to be done?

No, just adding the file should be enough.

File: android/app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources>
+    <string name="app_name">fheroes2</string>
+    <string name="activity_toolset_label">fh2 Werkzeugkasten</string>
```
Comment (oleg-derevenetz) [inline android/app/src/main/res/values-de/strings.xml:5]: How this label looks on the launcher screen? Isn't a bit long?

Comment (Laserlicht) [reply android/app/src/main/res/values-de/strings.xml:5]: Yes, it is shortened. But you can still see what is meant.

The question is, is that so bad? For me, about a third of the app labels are shortened. Also official ones from google.

grafik

Comment (oleg-derevenetz) [reply android/app/src/main/res/values-de/strings.xml:5]: Yes, that's true. For some reason, Google doesn't even allow to break icon labels into two lines instead shortening.

**Comment by ihhub** [comment]
@Laserlicht , thank you so much for this addition to the project!

**Pull request merged**